weather.py

```diff
@@ -24,7 +27,10 @@ def convert_date(iso_string):
     Returns:
         A date formatted like: Weekday Date Month Year e.g. Tuesday 06 July 2021
     """
-    pass
+    date = datetime(int(iso_string[0:4]),
+                    int(iso_string[5:7]),
+                    int(iso_string[8:10]))
```

Comment:
Nice use of string slicing to create the `datetime` object. P.S. there is also an inbuilt method `fromisoformat` which performs a similar action